api/lib/opentelemetry/baggage/builder.rb

```diff
@@ -6,13 +6,37 @@
 
 module OpenTelemetry
   module Baggage
-    # No op implementation of Baggage::Builder
+    # Operational implementation of Baggage::Builder
     class Builder
-      def set_value(key, value); end
+      attr_reader :entries
 
-      def remove_value(key); end
+      def initialize(entries)
```

Comment:
Should this be `@api private`? I think it is only supposed to be called from `Manager`.